### DISCUSSION TITLE
How to remove the card from the research page

### DISCUSSION THREAD

**Discussion by Srihitha-l** [greenelab/lab-website-template#204] [Q&A] [answered]
Currently, on the research and blog page, a card appears behind every post/citation.  I know that If I want to change how the card looks I can edit card.html and card.scss. But instead of changing the current card, I want to create another type of card for the citations and keep the current card for the blog. How do I remove the current card from behind the citations and add the new card?

**Comment by vincerubinetti** [comment, marked as answer]
There are three components that look similar: Card, Citation, and Post Excerpt. They all have border shadows by default, and thus may look like a "card component" in the general sense, but they are separate components in this template.

https://greenelab.github.io/lab-website-template/testbed#citation

So, the citations on the research page and blog posts on the blog page are already separate components, and you can style them separately, as you desire. Just edit `_styles/citation.scss` and `_styles/post-excerpt.scss`.

**Comment by Srihitha-l** [comment]
Thank You!
